cloud/operations.md

```diff
@@ -0,0 +1,50 @@
+# Operations
+Timescale Cloud provides an operations dashboard for managing your services. You
+can see the Operations dashboard in your Timescale Cloud account by navigating
+to the `Services` section, clicking the service you want to explore, and
+selecting the `Operations` tab.
+
+## General
+In the `General` section of the Operations dashboard, you can fork your service,
+reset the password, pause, or terminate the service.
+
+### Fork a service
+When you a fork a service, you create an exact copy of the service, including
+the underlying database. This allows you to create a copy that you can use for
+testing purposes, or to prepare for a major version upgrade.
```

Comment:
Do we want to emphasize that the forked service has its own `tsdbadmin` user password, different from the origin?

Reply:
That's a good point. We show users the "next steps" / "cheatsheet" page but if they skip it saving the password it isn't the same as the parent database (by design).